commands/basic/join.js

```diff
@@ -1,22 +1,23 @@
-const {prefix} = require("./../../config");
+const {prefix} = require("./../../config")
 
-join = (client) => {
+const join = (client) => {
     client.on("message", (message) => {
         if (!message.guild){
             return
         }
         if(message.author.bot){
             return
         }
-        msg = message.content.toLowerCase()
+        const msg = message.content.toLowerCase()
+        
         if(msg === prefix + "join" || msg === prefix + "j"){
```

Comment:
[time for regex](https://stackoverflow.com/a/2140723)?

Reply:
the top answer on the stackoverflow suggests using .toUpperCase() and comparing to another uppercase String? Which is what I do here except lower case. :(